Documentation/teaching/labs/device_drivers.rst

```diff
@@ -375,13 +375,13 @@ Access to the address space of the process
 ==========================================
 
 A driver for a device is the interface between an application and hardware. As
-a result, we often have to access a given user-space driver device. Accessing
-process address space can not be done directly (by de-referencing a user-space
-pointer). Direct access of a user-space pointer can lead to incorrect behavior
-(depending on architecture, a user-space pointer may not be valid or mapped to
-kernel-space), a kernel oops (the user-mode pointer can refer to a non-resident
-memory area) or security issues. Proper access to user-space data is done by
-calling the macros / functions below:
+a result, we often have to access a given user-space data. Accessing it can not
```

Comment:
Since we are doing changes here, could you also please remove the "a given". I think it sounds better without it, i.e. "As a result, we often have to access users-space data".

Reply:
Agreed, I will update so.

Reply:
Updated. :)